### ISSUE TITLE
Q: replicate remove command owltools

### ISSUE THREAD

**Issue by matentzn** [ontodev/robot#543]
Is it possible with the current version of ROBOT to replicate this old owltools command?

```
owltools ont.owl --merge-imports-closure --remove-axioms -t ObjectPropertyDomain --remove-axioms -t ObjectPropertyRange -t DisjointUnion -o out.owl
```

If not, I will just use owltools for this one for now; you can decide whether this is a strong enough use case ([monarch ontology](https://github.com/monarch-initiative/monarch-ontology)).

**Comment by rctauber** [comment]
I haven't used `owltools --remove-axioms` so let me check that I'm getting this correct. You would like to remove all `ObjectPropertyDomain` axioms and only the `ObjectPropertyRange` axioms that are disjoint unions?

**Comment by matentzn** [comment]
No I think the command is unnecessarily redundant; it just means:
remove all ObjectPropertyDomain, ObjectPropertyRange and DisjointUnionOf axioms... :) Thanks

**Comment by rctauber** [comment]
We created the `--axioms` values to make it easier for people to select the axioms without having a knowledge of OWL API. Originally, the `--axioms` values were to be the string names of the `AxiomType`: http://owlcs.github.io/owlapi/apidocs_5/org/semanticweb/owlapi/model/AxiomType.html

We could keep the simple names, but also add in the ability for more advanced users to use the `AxiomType` names, for example:
```
robot remove --input remove-test.owl \
  --axioms "ObjectPropertyRange ObjectPropertyDomain DisjointUnion" \
  --output after-test.owl
```

Would achieve what you're looking for, and it would be a very simple change to the code. @jamesaoverton - what are your thoughts on this?

**Comment by matentzn** [comment]
If you could easily extend the command that allows as parameters the AxiomType names that would be awesome!

**Comment by jamesaoverton** [comment]
Yes, let's add the exact AxiomType names.

**Comment by rctauber** [comment]
Resolved by #545 - feel free to reopen if you have any other problems with this, though.
